elastic/kibana
# [Security Solution][Alert Flyout] Enable network preview and previews in table tab

## Summary

This PR is a refactor of preview links in document details flyout:

1) Replace all the `EuiLink` component with a shared `PreviewLink` that renders a link based on field
2) Added IP (network) flyout previews through out document details flyout
3) Added preview capabilities for fields in table tab 

**Network previews**

<img width="1420" alt="image" src="https://github.com/user-attachments/assets/8dd52e1a-e48a-44cd-9eb3-bf34b067bf93">



**After**

https://github.com/user-attachments/assets/451292f6-3056-4362-87e4-e8170f0bda3f



**Exceptions**
IP addresses in entity details section are not yet worked on, as it is  owned by the Explore team and therefore has impacts outside of alerts flyout. 

<img width="919" alt="image" src="https://github.com/user-attachments/assets/4d5fb2be-b132-4db0-ae65-32caa8e0d49f">

**How to test**
- Using the normal resolver script should generate host, user and ip data for testing. 
- To test ip's in highlighted fields, user should create a custom rule and add some ip fields to highlighted fields

### Checklist
